docs/interacting-with-wpgraphql.md

```diff
@@ -0,0 +1,189 @@
+---
+uri: "/docs/interacting-with-wpgraphql/"
+title: "Interacting with WPGraphQL"
+---
+
+On this page, you will find details on the various ways you can interact with your WPGraphQL API.
+
+## GraphQL IDEs and Tools
+
+One of the most recognizable tools for interacting with GraphQL APIs is [GraphiQL](https://github.com/graphql/graphiql).
+
+GraphiQL is a React component that acts as an IDE for interacting with GraphQL APIs. While it might be the most recognizable GraphQL IDE tool (and is maintained by the GraphQL Foundation), it's just one of many tools you can use to interact with your WPGraphQL API.
+
+WPGraphQL ships with GraphiQL allowing you to search your GraphQL Schema and test GraphQL Queries and Mutations from your WordPress Admin.
+
+![Screenshot of GraphiQL in the WordPress Admin](./interacting-wordpress-admin-graphiql.png)
+
+**Below is a non-comprehensive list of helpful tools to interact with your WPGraphQL API:**
+
+- [WPGraphiQL](https://github.com/wp-graphql/wp-graphiql) – GraphQL IDE right in your WordPress dashboard
+- [GraphQL Playground](https://github.com/graphcool/graphql-playground) – GraphQL IDE that supports multi-column schema docs, tabs, query history, configuration of HTTP headers and GraphQL Subscriptions.
```

Comment:
Looks like this moved on Github to here: https://github.com/graphql/graphql-playground